### ISSUE TITLE
Function valued property erasing calls

### ISSUE THREAD

**Issue by Jand42** [fsharp/FSharp.Compiler.Service#534]
In this code:
```fsharp
    type TestFuncProp() =
        member this.Id = fun x -> x

    let wrong = TestFuncProp().Id 0 = 0
```
The expression body of `wrong` is just reported by FCS as `new TestFuncProp(()).get_Id(()) 0` (printed by `printExpr` in `ExprTests.fsx`). Even if `TestFuncProp().Id 0` is nested in multiple calls, it erases everything around it.

**Comment by dsyme** [comment]
This is probably the same as this one: https://github.com/fsharp/FSharp.Compiler.Service/issues/544

**Comment by dsyme** [comment]
Fixed by https://github.com/fsharp/FSharp.Compiler.Service/pull/549

**Comment by dsyme** [comment]
Fix is in 2.0.0.7
